code/modules/projectiles/updated_projectiles/magazines/specialist.dm

```diff
@@ -114,7 +114,7 @@
 	attack_self(mob/user)
 		if(current_rounds <= 0)
 			to_chat(user, "<span class='notice'>You begin taking apart the empty tube frame...</span>")
-			if(do_after(user,10, TRUE, 5, BUSY_ICON_BUILD))
```

Comment:
s the removal of the icon intended here?

Reply:
no visible message and the delay is minimal.